Automattic/simplenote-android
# Update Style Recreate

### Fix
Update the **_Style_** screen so that the scroll position is retained when the view is recreated by selecting a style.  See the animations below for illustration.

Before|After
-|-
<a href="https://user-images.githubusercontent.com/3827611/99319801-719ecc00-2827-11eb-883f-25ced493e988.gif"><img src="https://user-images.githubusercontent.com/3827611/99319801-719ecc00-2827-11eb-883f-25ced493e988.gif" width="300"></a>|<a href="https://user-images.githubusercontent.com/3827611/99319806-73688f80-2827-11eb-9596-c55588274633.gif"><img src="https://user-images.githubusercontent.com/3827611/99319806-73688f80-2827-11eb-9596-c55588274633.gif" width="300"></a>

### Test
1. Open navigation drawer.
2. Tap **_Settings_** item in drawer.
3. Tap **_Style_** setting under **_Appearance_** section.
4. Scroll any amount on **_Style_** screen.
5. Tap any style other than current selection on **_Style_** screen.
6. Notice style tapped is selected.
7. Notice scroll position is retained.

### Review
Only one developer and one designer are required to review these changes, but anyone can perform the review.

#### Note
Contact me for an installable build.

### Release
These changes do not require release notes.  `RELEASE-NOTES.txt` will be updated once the styles feature is complete.
